src/components/skeleton/index.tsx

```diff
@@ -19,7 +19,7 @@ export class SmoothlySkeleton {
 		const cssVariables = {
 			"--width": this.width,
 			"--distance": this.distance ?? "10rem",
-			...(this.color ? { "--color": this.color } : undefined),
+			...(this.color ? { "--text-color": this.color } : undefined),
```

Comment:
is this intended?

Reply:
no.

Reverting the file to its original state